### DISCUSSION TITLE
Welcome to cloud_native_application Discussions!

### DISCUSSION THREAD

**Discussion by kamarbaraka** [kamarbaraka/cloud_native_application#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samkombe** [comment]
what's this about?

**Reply by kamarbaraka** [reply]
is a practice in developing cloud-native applications

**Comment by samkombe** [comment]
what the goal of it?

**Reply by kamarbaraka** [reply]
to practice
